cli.js

```diff
@@ -29,32 +34,66 @@ var usage = `Usage
 Work in progress! Learn more at github.com/cabal-club
 `
 
-if (args.key) {
-  args.key = args.key.replace('cabal://', '').replace('cbl://', '').replace('dat://', '').replace(/\//g, '')
-  args.db = rootdir + args.key
+// Attempt to load local or homedir config file
+try {
+  var config
+  var configFilename = '.cabal.yml'
+  if (fs.existsSync(configFilename)) {
+    config = yaml.safeLoad(fs.readFileSync(configFilename, 'utf8'))
+  } else if (fs.existsSync(args.dir || homedir)) {
+    config = yaml.safeLoad(fs.readFileSync((args.dir || homedir) + '/' + configFilename, 'utf8'))
+  }
+  if (config && config.keys) {
+    keys = config.keys
```

Comment:
If you run `keys = config.keys.map(parseKey)` you'll let config files also use things like cabal:// cbl:// dat:// etc